setup.py

```diff
@@ -6,7 +6,7 @@
     packages=PEP420PackageFinder.find("src"),
     package_data={"cascade.executor": ["data/*.toml"]},
     package_dir={"": "src"},
-    install_requires=["numpy", "pandas", "scipy", "toml", "sqlalchemy"],
+    install_requires=["numpy", "pandas", "scipy", "toml", "sqlalchemy", "networkx"],
```

Comment:
This is fine. I suspect we'll end up wanting it for other things anyway. We're operating on a graph-like object.

Reply:
fixed